src/test/java/core/basesyntax/service/impl/WriterServiceImplTest.java

```diff
@@ -0,0 +1,54 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.TransactionException;
+import core.basesyntax.service.WriterService;
+import java.io.File;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class WriterServiceImplTest {
+    private static final String EXCEPTION_MESSAGE
+            = TransactionException.class.toString();
+    private static final String CORRECT_FILE_PATH = "src/main/resources/output.csv";
+    private static final String INCORRECT_FILE_PATH
+            = "C:/Users/.../my-project/src/main/resources/output.csv";
+    private static final String REPORT_CONTENT = "apple,120";
+    private static WriterService writerService;
```

Comment:
In general, it is more idiomatic to declare the service being tested as an instance variable rather than a static variable.